### PR TITLE
Improve code example in readme.md

### PR DESCRIPTION
**Pull request by toolforger**
Bug fix: It wouldn't display the "Press escape to exit" message initially (made me believe it's not working at all).
Code quality: move the FSArray to an initialization function so it can be called before loop and after pressing Space, making sure the program has the same clean slate in both situations.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -12,17 +12,27 @@ import sys
 from curtsies import FullscreenWindow, Input, FSArray
 from curtsies.fmtfuncs import red, bold, green, on_blue, yellow
 
+def setup_content:
```
Comment (thomasballinger) [inline readme.md:15]: looks like this should be removed

**Review by thomasballinger — COMMENTED** [review]
Thanks for fixing this, agreed it's much better to show this initially.

I don't think we should add the new function. Since the goal of the example is to demonstrate functionality (vs be useful) I like pressing space showing how to clear the screen by using a clean FSArray.

**Comment by toolforger** [comment]
I agree a function is a distraction, but the kind of duplicate code needed to make the example work as expected is a distraction, too.
How about removing the SPACE case? Removes the duplicate code, and it's not really necessary for an example: anybody who needs a clean scree can simply exit and restart the program.

File: readme.md (hunk 1 above)
Comment (toolforger) [reply readme.md:15]: Right, looks like a copy-paste error to me. It's not even valid Python (no indented line afterwards); I'm slightly surprised CI didn't catch this.
(Deleted my previous comment, I didn't see enough context and misinterpreted what you said.)

Comment (toolforger) [reply readme.md:15]: BTW I'm still wondering whether it isn't better to remove the SPACE case and inline the function.
Teaches one less useful pattern to deal with the "reinitialize the screen" case, but this is about showcasing the library, not about programming patterns. And having a function is a bit of a distraction (just like duplicate code would be).

Comment (thomasballinger) [reply readme.md:15]: I like that, yeah. I think I'll leave space for now.

**Comment by thomasballinger** [comment]
Accepted, manually merged. Thanks!

**Comment by thomasballinger** [comment]
https://github.com/bpython/curtsies/commit/1aa3f6ad8e4febeccfdd4c77c8e55143261ec996

**Pull request closed without merging**